core/src/test/java/org/apache/iceberg/TestCatalogUtil.java

```diff
@@ -218,14 +218,14 @@ public void loadCustomMetricsReporter_badClass() {
   public static class TestCatalog extends BaseMetastoreCatalog {
 
     private String catalogName;
-    private Map<String, String> flinkOptions;
+    private Map<String, String> options;
```

Comment:
[minor] should we call it `catalogProperties` like we do in glue in hive catalog imp ?